tests/system/large/test_remote_function.py

```diff
@@ -1800,3 +1800,63 @@ def float_parser(row):
         cleanup_remote_function_assets(
             session.bqclient, session.cloudfunctionsclient, float_parser_remote
         )
+
+
+@pytest.mark.parametrize(
+    ("memory_mib_args", "expected_memory"),
+    [
+        pytest.param({}, "1024Mi", id="no-set"),
+        pytest.param({"cloud_function_memory_mib": None}, "256M", id="set-None"),
```

Comment:
Why different values for unset vs None?

Reply:
unset should be backward compatible, meaning we should continue to set bigframes default of 1024M. `None` is a way to indicate that any explicit memory from the bigframes side should not be set, as added in the documentation "If you want to let the default memory of cloud functions be allocated, pass `None`". There is another param `cloud_function_timeout` with similar behavior. I acknowledge the confusion though, how about I send a separate PR to make unset and None behavior same in both param for consistency?